src/filtrering/filtrering-mine-filter/ny_mine-filter-rad.tsx

```diff
@@ -39,15 +40,15 @@ function NyMineFilterRad({filter, filtergruppe}: LagretFilterRadProps) {
         <div className="ny__mine-filter__rad">
             <Radio
                 className="ny__mine-filter__filternavn"
-                key={filter.filterId}
+                key={lagretFilter.filterId}
```

Comment:
I noticed this has `lagretFilter.filterId`, and veiledergrupper has `veiledergrupper.filterId`.. does that mean this should have the name `mineFilter.filterId`? :P

Reply:
I agree :P :)

Reply:
should I rename veilederGruppe to mineFilter here: https://github.com/navikt/veilarbportefoljeflatefs/blob/1a9c0105bc1a6dce29d408f4660ab895f61af8dc/src/filtrering/filtrering-veileder-grupper/ny_veileder_gruppe_rad.tsx#L14 ?

Reply:
Isn't this veiledergruppe, though?

Reply:
Yep, it is...

Reply:
so, leave `veilarbportefoljeflatefs/src/filtrering/filtrering-veileder-grupper/ny_veileder_gruppe_rad.tsx` like it is, but change this file. should be mineFilter, and not lagretFilter, since it's only a variable for this file, and this file is for mine filter.

Reply:
Yep, thats fine now.